mkdocs_static_i18n/reconfigure.py

```diff
@@ -611,8 +611,10 @@ def reconfigure_files(
                             log.debug(f"Use localized asset {i18n_file.locale} {i18n_file}")
 
             # theme (and overrides) files
-            elif self.is_default_language_build:
+            elif self.is_default_language_build or not file.src_uri.endswith(".md"):
```

Comment:
In the end I wonder if this is the same as:

```suggestion
            elif not file.is_documentation_page():
```

Reply:
Seems so, as the function asserts Markdown files and the is_default check is also redundant.
Most importantly works for my case ✌️